lib/evaluate.js

```diff
@@ -153,13 +153,13 @@ export const evaluate = async ({
   })
 
   const ignoredErrors = []
-
+  const honestMeasurements = measurements.filter(
+    m => m.taskingEvaluation === 'OK' && (m.consensusEvaluation === 'MINORITY_RESULT' || m.consensusEvaluation === 'MAJORITY_RESULT')
```

Comment:
I can see how this condition is a literal implementation of what I described in https://github.com/CheckerNetwork/spark-evaluate/issues/446.

This will leave out outlier measurements where the committee was too small or no majority was found. That's not desired.

I would like to include all measurements accepted by the tasking evaluation.

```suggestion
    m => m.taskingEvaluation === 'OK'
```

Reply:
Addressed in c6decc3